EasyPost/Utilities/Cryptography.cs

```diff
@@ -118,35 +121,41 @@ private static byte[] AsByteArray(this string str, Encoding? encoding = null)
         ///     Convert a byte array to a hex string.
         /// </summary>
         /// <param name="bytes">Byte array to convert to hex string.</param>
-        /// <returns>Hex string</returns>
-        private static string AsHexString(this byte[] bytes)
+        /// <returns>Hex string.</returns>
+        private static string AsHexString(this IReadOnlyList<byte> bytes)
         {
             // Fastest safe way to convert a byte array to hex string,
             // per https://stackoverflow.com/a/624379/13343799
-
             uint[] lookup32 = Lookup32;
-            char[] result = new char[bytes.Length * 2];
-            for (int i = 0; i < bytes.Length; i++)
+            char[] result = new char[bytes.Count * 2];
+            for (int i = 0; i < bytes.Count; i++)
```

Comment:
Does `Length` and `Count` actually do the same thing here?

Reply:
Yes. In fact, if I try to write `.Length`, my IDE automatically changes it to `.Count`:

Count is preferred, but could be mutable on a normal array, whereas Length gives a consistent result. Since this is an immutable list, Count == Length.
https://softwareengineering.stackexchange.com/a/294619